client/src/constants.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  PREFIX: 'na-rce-',
+  SOCKET: "http://localhost:9000",
```

Comment:
```suggestion
  SERVER_URL: "http://localhost:9000",
```
Let's call it `SERVER_URL`